.devcontainer/devcontainer.json

```diff
@@ -40,8 +40,8 @@
 			"version": "2.2.29"
 		},
 		"ghcr.io/devcontainers/features/terraform:1": {
-			"version": "1.0.3",
-			"terragrunt": "0.31.1"
+			"version": "1.6.2",
+			"terragrunt": "0.52.4"
```

Comment:
It would also be good here to bump the Terraform/Terragrunt versions used in the TF plan and apply workflows.